clash-prelude/src/Clash/Explicit/Signal.hs

```diff
@@ -194,7 +194,7 @@ module Clash.Explicit.Signal
   , fromEnable
   , enableGen
     -- * Clock
-  , Clock
+  , Clock(Clock)
```

Comment:
Do we really need to expose the constructor? It's not synthesizable is it?

Reply:
Huh, no this was unintentional. Will fix.